lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestContextStack.cs

```diff
@@ -203,10 +362,42 @@ public byte[] Call(byte[] memento)
             }
         }
```

Comment:
In the CloseContext handler, the close for the first context will be not received. You can verify then remove it from the test.